iroh/examples/transfer.rs

```diff
@@ -0,0 +1,324 @@
+use std::{
```

Comment:
This is not really an example but a bench, right?

Reply:
The intent is for it to be example that we also happen to use for netsim. It benches nothing on it's own though.